libs/ui/src/diagnostics/configuration_section.tsx

```diff
@@ -89,6 +92,16 @@ export function ConfigurationSection({
           {getBallotStyleIds(election, precinctSelection).join(', ')}
         </P>
       )}
+      {markThresholds?.definite && (
+        <P>
+          <SuccessIcon /> Mark Threshold: {markThresholds.definite}
+        </P>
+      )}
+      {markThresholds?.writeInTextArea && (
+        <P>
+          <SuccessIcon /> Write-in Threshold: {markThresholds.writeInTextArea}
```

Comment:
Maybe truncate these to 4 decimals? I don't expect mark thresholds will be very long decimals but doesn't hurt to be safe.